spec/systems/view_keyword_list_spec.rb

```diff
@@ -122,5 +125,44 @@
         end
       end
     end
+
+    it 'displays information link when background job completed' do
+      user = Fabricate(:user)
+      file_path = Rails.root.join('spec', 'fabricators', 'files', 'adword_keywords.csv')
+
+      visit keywords_path
+
+      within 'form' do
+        fill_in('username', with: user[:username])
+        fill_in('password', with: 'password')
+
+        click_button(I18n.t('auth.login'))
+      end
+
+      attach_file('csv_import_form[file]', file_path)
+
+      click_button(I18n.t('keyword.upload'))
```

Comment:
no need to test the upload feature again, this is the view keyword list test so we assume that the keyword is created somehow already, we just need to view the list.

so you can create the keyword in a database directly instead of upload it again.

Reply:
Fixed in cfd4e8eecf47b76d7e91c2445069f5f960a53c11